gnome-initial-setup/pages/language/cc-language-chooser.c

```diff
@@ -577,6 +577,7 @@ static void
 cc_language_chooser_finalize (GObject *object)
 {
 	CcLanguageChooser *chooser = CC_LANGUAGE_CHOOSER (object);
+    show_more (chooser);
```

Comment:
Agreed... Not really sure why this is needed, and the commit message does not make it clear.